src/patternfly/layouts/Grid/grid.scss

```diff
@@ -48,21 +49,43 @@
 
   > *,
   .pf-l-grid__item {
-    grid-column-start: var(--pf-l-grid__item--GridColumnStart);
-    grid-column-end: var(--pf-l-grid__item--GridColumnEnd);
+    order: var(--pf-l-grid--item--Order);
     min-width: 0;
     min-height: 0;
+    grid-column-start: var(--pf-l-grid__item--GridColumnStart);
+    grid-column-end: var(--pf-l-grid__item--GridColumnEnd);
+
+    // don't set base value for --pf-l-grid--item--Order as it defaults to 0
+    @media screen and (min-width: $pf-global--breakpoint--sm) {
+      order: var(--pf-l-grid--item--Order-on-sm, var(--pf-l-grid--item--Order));
+    }
+
+    @media screen and (min-width: $pf-global--breakpoint--md) {
+      order: var(--pf-l-grid--item--Order-on-md, var(--pf-l-grid--item--Order-on-sm, var(--pf-l-grid--item--Order)));
+    }
+
+    @media screen and (min-width: $pf-global--breakpoint--lg) {
+      order: var(--pf-l-grid--item--Order-on-lg, var(--pf-l-grid--item--Order-on-md, var(--pf-l-grid--item--Order-on-sm, var(--pf-l-grid--item--Order))));
+    }
+
+    @media screen and (min-width: $pf-global--breakpoint--xl) {
+      order: var(--pf-l-grid--item--Order-on-xl, var(--pf-l-grid--item--Order-on-lg, var(--pf-l-grid--item--Order-on-md, var(--pf-l-grid--item--Order-on-sm, var(--pf-l-grid--item--Order)))));
+    }
+
+    @media screen and (min-width: $pf-global--breakpoint--2xl) {
+      order: var(--pf-l-grid--item--Order-on-2xl, var(--pf-l-grid--item--Order-on-xl, var(--pf-l-grid--item--Order-on-lg, var(--pf-l-grid--item--Order-on-md, var(--pf-l-grid--item--Order-on-sm, var(--pf-l-grid--item--Order))))));
+    }
   }
 
   // Loop through $breakpoints map to generate responsive classes
-  @each $breakpoint, $value in $pf-global--breakpoint-map {
+  @each $breakpoint, $value in $pf-l-grid--breakpoint-map {
```

Comment:
I think this might be a breaking change? I've seen instances where products are overwriting or building on our existing maps for custom entries.